package.json

```diff
@@ -7,12 +7,13 @@
   },
   "dependencies": {
     "bokehjs": "0.12.9",
-    "check-dependencies": "1.1.0",
+    "check-dependencies": "^1.1.0",
     "classnames": "2.2.6",
     "css-loader": "0.28.4",
     "exports-loader": "0.6.4",
     "imports-loader": "0.7.1",
     "moment": "2.20.1",
+    "moment-timezone": "^0.5.25",
```

Comment:
Does this pass the linting test for you?  I thought we force pinned versions.

Reply:
Yeah, the linter should have caught this, but just remove the caret to pin the versions.

Reply:
Fixed!